LambdaRuntimeDockerfiles/SmokeTests/test/ImageFunction.SmokeTests/ImageFunctionTests.cs

```diff
@@ -67,14 +66,14 @@ public class ImageFunctionTests : IDisposable
 
         public ImageFunctionTests()
         {
-            _executionRoleName = $"{TestIdentifier}-{Guid.NewGuid()}";
             _functionName = $"{TestIdentifier}-{Guid.NewGuid()}";
             var lambdaConfig = new AmazonLambdaConfig()
             {
                 RegionEndpoint = TestRegion
             };
             _lambdaClient = new AmazonLambdaClient(lambdaConfig);
             _iamClient = new AmazonIdentityManagementServiceClient(TestRegion);
+            _executionRoleArn = Environment.GetEnvironmentVariable("AWS_LAMBDA_SMOKETESTS_LAMBDA_ROLE");
```

Comment:
nit: Should we `Assert.NotNull` like the other environment variable? Not sure how this fails if it isn't set.

Reply:
Thanks, I'd rather fail fast then attempt to create things and fail.